modules/server/src/main/scala/higherkindness/compendium/http/RootService.scala

```diff
@@ -27,6 +27,7 @@ import org.http4s.HttpRoutes
 import org.http4s.circe.CirceEntityCodec._
 import org.http4s.dsl.Http4sDsl
 import org.http4s.headers.Location
+import higherkindness.compendium.core.refinements._
```

Comment:
:sweat_smile: Please, order `import...` statements :wink:

Reply:
I forgot about that import, fixing

Reply:
There is scalafmt conf for that IIRC. Should we use it?